src/VersionProvider.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+class VersionProvider
+{
+    public function __construct(public $version = '3.0')
```

Comment:
Maybe add a method for possibility of getting it dynamically? Or it's redundant for now?